tests/unittests/GraphOptzTest.cpp

```diff
@@ -1817,6 +1825,86 @@ TEST_F(GraphOptz, foldQuantizeIntoVarMultipleUsages) {
   }
 }
 
+static SaveNode *getUniqueSaveNode(Function *F) {
+  SaveNode *foundSaveNode = nullptr;
+  for (auto &node : F->getNodes()) {
+    if (auto *s = llvm::dyn_cast<SaveNode>(&node)) {
+      EXPECT_EQ(foundSaveNode, nullptr);
+      foundSaveNode = s;
+    }
+  }
+  EXPECT_NE(foundSaveNode, nullptr);
```

Comment:
nit: `ASSERT_NE` instead? The caller is expecting this to be non-null, so could cause a segfault in the test otherwise.

Reply:
`ASSERT` only work in functions returning void or in the main test functions. If I use it, it won't compile.
```
error: void value not ignored as it ought to be
 2034 |   ASSERT_NE(foundSaveNode, nullptr);
```